frontend/public/components/app-contents.tsx

```diff
@@ -201,6 +201,520 @@ const AppContents: React.FC<{}> = () => {
+      <LazyRoute
+        path="/settings/idp/gitlab"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/gitlab-idp-form' /* webpackChunkName: "gitlab-idp-form" */
+          ).then((m) => m.AddGitLabPage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/google"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/google-idp-form' /* webpackChunkName: "google-idp-form" */
+          ).then((m) => m.AddGooglePage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/htpasswd"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/htpasswd-idp-form' /* webpackChunkName: "htpasswd-idp-form" */
+          ).then((m) => m.AddHTPasswdPage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/keystone"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/keystone-idp-form' /* webpackChunkName: "keystone-idp-form" */
+          ).then((m) => m.AddKeystonePage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/ldap"
+        exact
+        loader={() =>
+          import('./cluster-settings/ldap-idp-form' /* webpackChunkName: "ldap-idp-form" */).then(
+            (m) => m.AddLDAPPage,
+          )
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/oidconnect"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/openid-idp-form' /* webpackChunkName: "openid-idp-form" */
+          ).then((m) => m.AddOpenIDIDPPage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/basicauth"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/basicauth-idp-form' /* webpackChunkName: "basicauth-idp-form" */
+          ).then((m) => m.AddBasicAuthPage)
+        }
+      />
+      <LazyRoute
+        path="/settings/idp/requestheader"
+        exact
+        loader={() =>
+          import(
+            './cluster-settings/request-header-idp-form' /* webpackChunkName: "request-header-idp-form" */
+          ).then((m) => m.AddRequestHeaderPage)
+        }
+      />
+      <LazyRoute
+        path="/settings/cluster"
+        loader={() =>
+          import(
+            './cluster-settings/cluster-settings' /* webpackChunkName: "cluster-settings" */
+          ).then((m) => m.ClusterSettingsPage)
+        }
+      />
+
+      <LazyRoute
+        path={'/k8s/cluster/storageclasses/~new/form'}
+        exact
+        loader={() =>
+          import('./storage-class-form' /* webpackChunkName: "storage-class-form" */).then(
+            (m) => m.StorageClassForm,
+          )
+        }
+      />
+
+      <Route path="/k8s/cluster/:plural" exact component={ResourceListPage} />
+      <Route path="/k8s/cluster/:plural/~new" exact component={CreateResource} />
+      <Route path="/k8s/cluster/:plural/:name" component={ResourceDetailsPage} />
+      <LazyRoute
```

Comment:
@karthikjeeyar - FYI: this change deleted the route for the new 'debug pod' feature.  I have created a bug https://bugzilla.redhat.com/show_bug.cgi?id=2053304 and fix at https://github.com/openshift/console/pull/11040

Reply:
I am extremely sorry about that, not sure how this line got removed (could be due a rebase) and Thanks for finding and fixing it.